src/world/Macros/SummonMacros.hpp

```diff
@@ -0,0 +1,8 @@
+/*
+Copyright (c) 2014-2020 AscEmu Team <http://www.ascemu.org>
+This file is released under the MIT license. See README-MIT for more information.
+*/
+
+#pragma once
+
+#define SUMMON_SLOTS 6 // ?UNIT_SUMMON_SLOTS?
```

Comment:
You can remove this macro and use UNIT_SUMMON_SLOTS

Reply:
thanks, I thought so ,🤗